SemanticAutocomplete.md

```diff
@@ -0,0 +1,127 @@
+Semantic Autocomplete V2
+========================
+
+#### Table of Contents
+
+- [Summary](#summary)
+- [Request](#request)
+- [Response](#response)
+- [Example](#example)
+- [Versioning](#versioning)
+
+
+## Summary
+The autocomplete service wraps the [Solr Suggester](https://lucene.apache.org/solr/guide/6_6/suggester.html) which is configured in DSAD's Semantic Search Solr boxes. Version 2 of the Semantic Autocomplete service is a complete rewrite of the original service and makes use of the latest FSTs provided to DSAD from Textkernel which are updated sparodically throughout the year. With the exception of some removed feilds from the response, this version of the service functions exactly like V1.
```

Comment:
I would say 'The Semantic Autocomplete service wraps ...' We have multiple autocomplete services after all.

Reply:
'sporadically' but I wouldn't use that term. It has kind of a negative connotation.